spec/models/miq_report/seeding_spec.rb

```diff
@@ -1,5 +1,5 @@
 describe MiqReport do
   context "Seeding" do
-    include_examples(".seed called multiple times", 144)
+    include_examples(".seed called multiple times", 145)
   end
 end
```

Comment:
Ugh, again with these counting specs...

Reply:
You :clap: can :clap: never :clap: stop :clap: counting!!